packages/dds/tree/src/test/simple-tree/toMapTree.spec.ts

```diff
@@ -260,12 +260,49 @@ describe("toMapTree", () => {
 				["b", "Hello world"],
 				["c", null],
 				["d", undefined as unknown as InsertableContent], // Should be skipped in output
-				["e", { age: 37, name: "Jill" }],
 			];
 			const tree = new Map<string, InsertableContent>(entries);
 
 			const actual = nodeDataToMapTree(tree, [schema]);
 
+			const expected: MapTree = {
+				type: brand("test.map"),
+				fields: new Map<FieldKey, MapTree[]>([
+					[brand("a"), [{ type: leaf.number.name, value: 42, fields: new Map() }]],
+					[
+						brand("b"),
+						[{ type: leaf.string.name, value: "Hello world", fields: new Map() }],
+					],
+					[brand("c"), [{ type: brand(leaf.null.name), value: null, fields: new Map() }]],
+				]),
+			};
+
+			assert.deepEqual(actual, expected);
+		});
+
+		it("Non-empty map (input type: Record)", () => {
+			const schemaFactory = new SchemaFactory("test");
+			const childObjectSchema = schemaFactory.object("child-object", {
```

Comment:
I'd prefer tests to use the encouraged pattern of declaring a named class here unless you have a reason not to. Its a better developer experience (better IntelliSense, easier to refer to types, more extensible, works with recursive types etc)

Reply:
Reverted this test for now anyways.